packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -333,6 +334,24 @@ class ResizeHandle extends (Widget as any) {
       );
     }
 
+    eventsEngine.on(
```

Comment:
Why do we need to maintain this subscription in scenarios where the double-click functionality is not operational? For instance, when only the resize functionality is being utilized, without any collapsing features.